docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -108,6 +108,7 @@
     "import cudf\n",
     "import cupy\n",
     "import geopandas\n",
+    "import geodatasets\n",
```

Comment:
Since we directly download datasets from aws, we don't really depend on geodatasets here.